### PR TITLE
[Do Not Merge] SAP Hana backup support

### PR DESCRIPTION
**Pull request by siddharth7**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/backup/_format.py [hunk 1]
```diff
@@ -48,6 +84,39 @@ def transform_job(result):
...
+    return OrderedDict([('Name', result['name']),
+                        ('Resource Group', result['resourceGroup']),
+                        ('BackupManagementType', result['properties']['backupManagementType']),
+                        ('WorkloadType', result['properties']['workLoadType'])])
+
+
+def transform_workload_policy_show(result):
+    columns = []
+    columns.append(('Name', result['name']))
+    columns.append(('WorkloadType', result['properties']['workLoadType']))
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/backup/_format.py]:
```
columns = [
    ('Name', result['name']),
    ('WorkloadType', result['properties']['workLoadType'])
]
```
Maybe more readable

**Comment by yonzhan** [comment]
@qwordy please help deal with this PR.

**Comment by qwordy** [comment]
Some checks were not successful. Remember to fix.

**Comment by MyronFanQiu** [comment]
@siddharth7 Hello. Since it's a huge change, did you schedule a quick meeting before with any one of CLI team? We can have a quick meeting if you want.

**Comment by MyronFanQiu** [comment]
Close it since a new pr appears. https://github.com/Azure/azure-cli/pull/11066

**Pull request closed without merging**